utils/shoppingActions.ts

```diff
@@ -122,6 +122,10 @@ export const getActionButtonColor = (action: ShoppingActions): string => {
   return color
 }
 
+export const getActionButtonLabel = (action: ShoppingActions): string => {
+  return action.replace(/_/g, ' ')
+}
```

Comment:
How about making a map to proxy action to a translation key ?
If so, every action will have a more readable label in any language.

Reply:
so I did this, seems to work fine